java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/core/DbzCdcEventConsumer.java

```diff
@@ -76,6 +83,22 @@ public void handleBatch(
             DebeziumEngine.RecordCommitter<ChangeEvent<SourceRecord, SourceRecord>> committer)
             throws InterruptedException {
         var respBuilder = GetEventStreamResponse.newBuilder();
+        // wake up the thread blocked in `DbzCdcEngineRunner::start()`
+        startSignal.countDown();
+        if (!isStarted.get()) {
+            isStarted.set(true);
```

Comment:
`isStarted` is a atomic. It seems to suggest that there can be multiple thread concurrently calling `handleBatch`. If that is the case, I think we should use `compareAndSet` instead.

Reply:
The `handleBatch` will only be called in a dedicated thread.

Reply:
Then why do we need `AtomicBoolean` here? I think we should either use `Boolean` if it is executed always in the same thread or use `compareAndSet` if atomicity is needed.